### DISCUSSION TITLE
Conda install problem:  qt.qpa.plugin: Could not load the Qt platform plugin "xcb"

### DISCUSSION THREAD

**Discussion by Chaminu** [wavespectra/wavespectra#133] [General]
I installed v4.0.3 using conda on Rocky Linux 9.4.
Tried a simple test to readww3 and plot Hs and got an error:
_qt.qpa.plugin: Could not load the Qt platform plugin "xcb"_
I am not sure if this is my environment 
Fixed by:
**export QT_QPA_PLATFORM=offscreen**

**Comment by rafa-guedes** [comment]
@Chaminu thanks for reporting this and leaving the fix. I haven't had this sort of issue with qt on ubuntu.
